src/components/UserMenu/UserMenu.tsx

```diff
@@ -33,14 +34,26 @@ export const UserMenu = () => {
     authService.clearStorage();
   };
 
+  if (loading) {
```

Comment:
For me it candidate to create reusable component
`<Preloader isLoading={isLoading} error={error}>
.....
<Preloader />`
If it is not clear let me know in chat